src/functionaltest/java/com/ericsson/ei/frontend/SubscriptionHandlingFunctionality.java

```diff
@@ -0,0 +1,128 @@
+package com.ericsson.ei.frontend;
+
+import org.junit.*;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+
+import com.ericsson.ei.config.SeleniumConfig;
+import com.ericsson.ei.frontend.pageobjects.IndexPage;
+import com.ericsson.ei.frontend.pageobjects.SubscriptionPage;
+
+public class SubscriptionHandlingFunctionality extends SeleniumBaseClass {
+
+    private static final String DOWNLOADEDTEMPLATEFILEPATH = String.join(File.separator,
```

Comment:
Using _ between caps in variables make them a lot easier to read at first glance, like: DOWNLOADED_TEMPLATE_FILE_PATH

Reply:
true..